cores/esp8266/spiffs_api.cpp

```diff
@@ -127,6 +127,12 @@ bool isSpiffsFilenameValid(const char* name)
 
 }; // namespace
 
+extern "C" void spiffs_weak_end(void)
```

Comment:
Function name is a bit misleading: this function is not weak. 
I was going to suggest just using `spiffs_end()` and `littlefs_end()` across the board, but I find it does make sense to have the name reflect the weak symbol trick. 

What do you think of something like:
`spiffs_end_weaklink()`
?

Reply:
It is for internal call only right?
Then it does indeed need some reference to the weak trick.

Reply:
Functions are renamed, indeed `weak` should not appear in their name.
A link to the snippet above is added in source code.